### Problem Statement 1:
Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an effect or not.

Population Mean = 100

Population Standard Deviation = 15

Sample Size = 36

Sample Mean = 108

H0: μ=100 The diet will not have an effect on blood glucose levels

H1: μ>100 The diet will have a positive effect on blood glucose levels

Since n>30 and we have the population's standard deviation, we will perform a one tail z test

In [2]:
from math import sqrt

population_mean = 100
population_sd = 15
n=36
sample_mean = 108

z_score = (sample_mean-population_mean)/(population_sd/sqrt(n))
z_score

3.2

At 5% significance level the z_score is 1.64. Since the computed z_score is outside our acceptable region the null hypothesis is rejected.

### Problem Statement 2:
In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second
state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple
random sample of 100 voters are surveyed from each state.
What is the probability that the survey will show a greater percentage of Republican
voters in the second state than in the first state?

P1 = Proportion of the voters who are Republicans in State 1

P2 = Proportion of the voters who are Republicans in State 2

p1 = Sample Proportion of the voters who are Republicans in State 1

p2 = Sample Proportion of the voters who are Republicans in State 1

H0: The proportion of voters would remain the same in the sample

H1: The proportion of voters would be different

Two Tail Z test since n>30

In [4]:
P1 = 0.52
p1 = 0.52
n1 = 100
P2 = 0.47
p2 = 0.47
n2 = 100
pooled_sample_proportion = ((P1*n1) + (P2*n2))/(n1+n2)
standard_error = sqrt(pooled_sample_proportion*(1-pooled_sample_proportion)*((1/n1)+(1/n2)))
z = (p1-p2)/standard_error
z

0.7071421391774789

The p-value of our computed test statistic is 0.7580. Hence, the probability that the survey will show a greater percentage of Republican voters in the second state than in the the first state is 0.242 (1 - 0.758).

### Problem Statement 3:
You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard
deviation is 209. How well did you score on the test compared to the average test taker?

In [8]:
z = (1100-1026)/209
z

0.35406698564593303

Scored better than 63.68% of test takers with 0.35 standard deviation above the mean.